package.json

```diff
@@ -59,7 +69,7 @@
     "@esri/hub-search": "^9.32.1",
     "@esri/hub-sites": "^9.32.1",
     "@esri/hub-teams": "^9.32.1",
-    "config": "^3.3.6",
+    "coverage-badges-cli": "^1.0.11",
```

Comment:
This should be installed this as a dev dependency;  that prevents its installation in the deployment which doesn't require the library.

Reply:
Added coverage badge package as dev dependency. I might have to  do this other Koop plugins as well. Will make a note to do this in my new PR for those repos.